### ISSUE TITLE
Spansh data

### ISSUE THREAD

**Issue by Tromador** [eyeonus/EDDBlink-listener#25]
Now we have a plugin which gets us regular data, can we update the listener to use that, so we can reinstate the server end?

I am presuming (hoping) it's pretty trivial changes to the listener to use the new plugin to get that data inserted.

Then with good data hopefully we can get the other db issues etc and so on fixed up, maybe rope in others to assist.

**Comment by eyeonus** [comment]
Yes, just have to modify the check_update() method to use use the spansh plugin instead. I'll get around to it shortly.

**Comment by Tromador** [comment]
Long since completed.
